engine/src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -110,11 +101,11 @@ object CromwellTestkitSpec {
   }
 
   /**
-   * Akka TestKit appears to be unable to match errors generated by `log.error(Throwable, String)` with the normal
-   * `EventFilter.error(...).intercept {...}` mechanism since `EventFilter.error` forces the use of a dummy exception
-   * that never matches a real exception.  This method works around that problem by building an `ErrorFilter` more
-   * explicitly to allow the caller to specify a `Throwable` class.
-   */
+    * Akka TestKit appears to be unable to match errors generated by `log.error(Throwable, String)` with the normal
```

Comment:
This diff might have happened due to the cmd + alt + L. Will revert it :confused: